<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>materialPage</class>
 <widget class="MaterialPage" name="materialPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Kopie:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="copyBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Schl.-Nr.:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editKey"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Bezeichnung:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PageTextEdit" name="editDescr">
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="4">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Einheit:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editUnit"/>
       </item>
      </layout>
     </item>
     <item row="0" column="4">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Minuten</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editMinutes"/>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Netto:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editNetto"/>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Brutto:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editBrutto"/>
       </item>
      </layout>
     </item>
     <item row="0" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>EKP:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editEkp"/>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_8"/>
     </item>
     <item row="2" column="4">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Verarb.-Preis:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTotal">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout_10"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaterialPage</class>
   <extends>QWidget</extends>
   <header>pages/material_page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PageTextEdit</class>
   <extends>QTextEdit</extends>
   <header>functionality/utils.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>copyBox</tabstop>
  <tabstop>editKey</tabstop>
  <tabstop>editDescr</tabstop>
  <tabstop>editNetto</tabstop>
  <tabstop>editBrutto</tabstop>
  <tabstop>editEkp</tabstop>
  <tabstop>editMinutes</tabstop>
  <tabstop>editUnit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>materialPage</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>materialPage</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copyBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>materialPage</receiver>
   <slot>CopyData(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>CopyData(QString)</slot>
 </slots>
</ui>
